src/schema/nmdc.yaml

```diff
@@ -439,6 +439,7 @@ classes:
     slots:
       - applies to person
       - applied role
+      - applied roles
```

Comment:
so there will be two fields, `applied role` and `applied roles`? Seems a little unusual. However, go ahead and merge if you think this will work we can always revise on a future iteration

Reply:
also you have the plural form set to required: true, so there will always be a list of length 1, as well as the single valued?

Reply:
I set the required to `false` for `applied role` (singular). I only left the `applied role` slot there b/c I didn't want to invalidate the data currently in the database that uses this slot. After the databases is updated using the `applied roles` slot, I will remove it from the `credit association`.